packages/components/src/public/SchemaQuery.ts

```diff
@@ -105,8 +105,12 @@ export class SchemaQuery extends Record({
         };
     }
 
-    static createAppSelectionKey(targetSQ: SchemaQuery, keys: any[]): string {
-        return [APP_SELECTION_PREFIX, resolveSchemaQuery(targetSQ), keys.join(';')].join('|');
+    static createAppSelectionKey(targetSQ: SchemaQuery, prefix?: string, keys?: any[]): string {
```

Comment:
If you add the new prefix argument after the keys argument you won't need to alter the existing usage of this method.

Reply:
Also, it looks like this argument was added so that `QueryModel` and `QueryGridModel` will create the same `selectionKeys`, but `QueryGridModel` selection keys actually come from `getStateModelId`. QueryGridModel only uses `createAppSelectionKey` for models that have a `keyValue`, but that usage is pretty suspect because I doubt we're doing anything with selections for models that have a `keyValue` because those models only ever have a single row.

Reply:
Thanks Alan, I missed that getStateModelId function.  I reverted changes here and am now using that function for the model Id